2.1_first_module.ipynb

```diff
@@ -242,16 +233,27 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "val test10result = ???\n",
+    "// Test with width 10\n",
```

Comment:
We don't want to leave these as exercises for the student?

Reply:
Yes, not sure how that got changed, good catch

Reply:
@edwardcwang I fixed this up. Now uses have to add their own code here. This required changing runtest.py to handle the error generated by this.